pysmt/shortcuts.py

```diff
@@ -226,6 +226,14 @@ def Solver(quantified=False, name=None, logic=None):
                                     name=name,
                                     logic=logic)
 
```

Comment:
We have a shortcut get_model. Is there a good interface for a similar shortcut for unsat cores? (get_unsat_core)?